cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -473,7 +474,7 @@ public ClusterModel clusterModel(long from,
                                    ModelCompletenessRequirements requirements,
                                    boolean populateReplicaPlacementInfo,
                                    OperationProgress operationProgress)
-      throws NotEnoughValidWindowsException {
+      throws NotEnoughValidWindowsException, TimeoutException {
```

Comment:
If we intend to propagate `TimeoutException` all the way to higher layers, can we update the JavaDoc as we did for `NotEnoughValidWindowsException` for consistency? Alternatively, would it be cleaner to return `null` if the cluster model creation has failed due to not being able to resolve the cluster capacity, and handle the null cluster capacity response?

Reply:
I prefer the first approach. Because if the cluster model is unable to generate, there is nothing CC should do other than notify the user about the error.